module/extension/Director/src/Director.cpp

```diff
@@ -23,6 +23,9 @@
 
 namespace module::extension {
 
+    using component::DirectorTask;
```

Comment:
I think rather than component, director might make more sense (then you can also just name it `director::Task`)

Reply:
there's already a director folder with implementation for some director functions
it seems not right to put the task inside that folder and add a director namespace when everything else isn't

Reply:
True